### PR TITLE
SDK-2936: PYZ prefix issues

### PR DESCRIPTION
**Pull request by sergeyspryker**
## PR Description
https://spryker.atlassian.net/browse/SDK-2935

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by DmytroKlymanSpryker — APPROVED** [review]

File: docs/scu/dev/troubleshooting/pyz_prefix_issues.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+4. Remove constants that override the same parent constant with the same value.
+
+### Rename transfers
+
+1. In the `shared` folder, replace the following for `*.xml` transfers files:
+   * `"(pyz)(.)(\w*)` with `"\L$2\E$3`
+   * `"(Pyz)(.)(\w*)` with `"$2E$3`
+
+2. In `*.php` files, replace regexp `Transfer\->(get|set|require)Pyz` with `Transfer->$1`.
+3. In `*.twig` files, replace transfer getters `.getPyz` or `.pyz`
```
Comment (andriitserkovnyi) [inline docs/scu/dev/troubleshooting/pyz_prefix_issues.md:34]: replace with what?

File: docs/scu/dev/troubleshooting/pyz_prefix_issues.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+### DB schema renaming
+
+It's important to note that all the custom tables should be with `pyz_` prefixes due to Spryker conventions
+
+1. Replace in `*.xml` `pyz_`
```
Comment (andriitserkovnyi) [inline docs/scu/dev/troubleshooting/pyz_prefix_issues.md:44]: Replace with what?

File: docs/scu/dev/troubleshooting/pyz_prefix_issues.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Replace regexp `on (get|set|add|map)Pyz` with `on $1`.
+2. Replace regexp `\->(get|set|add|map)Pyz` with `->$1`.
+3. Replace `class Pyz` with `class`.
+4. Replace `BasePyz` with `Base`.
+5. Replace `([a-z])Pyz([A-Z])` with `$1$2`.
+
+### Rename variables and class names
+
+1. Search and replace by `\w\\Pyz` (regexp), `Pyz(`, `: Pyz`, `(Pyz`
```
Comment (andriitserkovnyi) [inline docs/scu/dev/troubleshooting/pyz_prefix_issues.md:56]: Replace with what?

File: docs/scu/dev/troubleshooting/pyz_prefix_issues.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+2. Check and replace all the  `pyz` occurrences by regexp `(?<!(namespace |use |@[a-z]+ \\))(Pyz|pyz)`
```
Comment (andriitserkovnyi) [inline docs/scu/dev/troubleshooting/pyz_prefix_issues.md:57]: Same

**Pull request merged**